platforms/jvm/ear/build.gradle.kts

```diff
@@ -14,17 +14,16 @@ errorprone {
 dependencies {
     api(libs.groovy)
     api(libs.inject)
-    api(libs.jsr305)
 
     api(projects.baseServices)
+    api(projects.core)
     api(projects.coreApi)
     api(projects.languageJvm)
     api(projects.modelCore)
     api(projects.platformJvm)
 
     implementation(projects.serviceLookup)
     implementation(projects.stdlibJavaExtensions)
-    implementation(projects.core)
     implementation(projects.fileOperations)
```

Comment:
Why is this necessary?

Reply:
Reverted that. Project health showed this as a problem, but it seems it's not a problem anymore after rebase